WordPress/wp-feature-notifications
# Docs/storybook controls

## What?
In the first storybook implementation some controls were completely absent.

## How?
as [explained here](https://storybook.js.org/docs/react/writing-stories/args) i've provided some custom controls to the components, then linked the date picker

## Testing Instructions
Start the storybook with `npm run start:storybook` then check the paths
[wp-feature-notifications-notification-hub-single--single](http://localhost:6006/?path=/story/wp-feature-notifications-notification-hub-single--single)
[wp-feature-notifications-dashboard-single--single](http://localhost:6006/?path=/story/wp-feature-notifications-dashboard-single--single)

![storybook](https://user-images.githubusercontent.com/8550908/232309905-c6d71870-7587-45be-aea2-cc62f7ffa55d.gif)
